debug_module.tex

```diff
@@ -217,6 +217,8 @@ \subsection{Abstract Command Listing}
         \hline
         1 & Quick Access & \pageref{quick access} \\
         \hline
+        2 & Access Memory Command & \pageref{access memory} \\
```

Comment:
should we call this something like "Access Memory from Hart Perspective" (or a more catchy phrase?)

Reply:
I think this is fine here, but I do agree that in other places we need to be more explicit about what's going on.